docs/konflux/release_operator_with_konflux.md

```diff
@@ -499,13 +507,11 @@ Results:
   configuration] to include additional registry prefixes.
 ```
 
-
-### Unable to pull images from staging environment
-The images have a default pullspec [reference](https://github.com/parodos-dev/orchestrator-helm-operator/blob/9e2371748bfbe899f945e9a5655622483504e660/bundle.konflux.Dockerfile#L12) that points to the production registry. When deploying from the staging IIB catalog, the cluster will try to pull the images from the production registry as defined in the spec and fail. To fix this issue, the cluster has to be configured so that it is aware of registry mirrors for specific image pullspecs. Configure your cluster by deploying this [manifest](https://github.com/parodos-dev/orchestrator-helm-operator/blob/main/docs/konflux/imagedigestmirrorset.yaml) that will instruct the cluster to fallback to the staging registry when the images are not in production.
-
+### Cluster fails to pull images from production registry (registry.redhat.io) because images are only in staging
+Deploy this [ImageDigestMirrorSet](imagedigestmirrorset.yaml) to your cluster to configure the cluster to use the staging registry as mirror to the production registry.
 
 ## Command tips
 * Retrieve the components for a given application:
 ```console
-oc get components -ojsonpath='{range .items[?(@.spec.application=="helm-operator")]}{.metadata.name}{"\n"}{end}}'
+oc get components -ojsonpath='{range .items[?(@.spec.application=="helm-operator-1-2")]}{.metadata.name}{"\n"}{end}}'
```

Comment:
Same here: -1-3?

Reply:
I'd leave it as it is since it's an example.